eval/asserts/boundary-clean-output.mjs

```
@@ -0,0 +1,146 @@
+  return content
+    .split('|')
+    .slice(1, content.endsWith('|') ? -1 : undefined)
+    .map(stripLightMarkdown);
+}
+
+function isSelectedRejectValue(value) {
+  if (/\b(?:APPROVE\s*\/\s*REJECT|REJECT\s*\/\s*APPROVE)\b/i.test(value)) {
+    return false;
+  }
+
+  return /^REJECT(?:[.!。！？]*|\s*(?:\([^()\r\n]*\)|（[^（）\r\n]*）|\[[^\[\]\r\n]*\]|[-–—]\s*\S.*))$/i.test(value);
```

ESLint

[error] Unnecessary escape character: \[.

(no-useless-escape)